frontend/public/components/resource-pages.ts

```diff
@@ -169,6 +169,7 @@ export const resourceListPages = ImmutableMap<GroupVersionKind | string, () => P
   .set(referenceForModel(StorageClassModel), () => import('./storage-class' /* webpackChunkName: "storage-class" */).then(m => m.StorageClassPage))
   .set(referenceForModel(CustomResourceDefinitionModel), () => import('./custom-resource-definition' /* webpackChunkName: "custom-resource-definition" */).then(m => m.CustomResourceDefinitionsPage))
   .set(referenceForModel(VirtualMachineModel), () => import('../kubevirt/components/vm' /* webpackChunkName: "virtual-machines" */).then(m => m.VirtualMachinesPage))
+  .set(referenceForModel(VmTemplateModel), () => import('../kubevirt/components/vm-template' /* webpackChunkName: "vm-templates" */).then(m => m.VirtualMachineTemplatesPage))
```

Comment:
There was some issue about the TemplateModel being usable once someone decides to have a dedicated page for that? @rawagner Wasn't it? I forgot about the details..

Reply:
Right. I played with the code a little a found a solution which will use resource-pages and wont break if someone creates a page for general templates. See my other comments